### PR TITLE
Add a code of conduct for the NVDA project adopted by NV Access

### PR DESCRIPTION
**Pull request by michaelDCurran**
This adds NV Access's official code of conduct for the NVDA project. This is also linked to from the readme and issue / pr templates.

### CODE REVIEW

**Review by Qchristensen — APPROVED** [review]
Looks good, thanks everyone.

**Review by JamesBoreham — APPROVED** [review]

**Review by feerrenrut — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## 3. Expected Behaviour
+The following behaviours are expected and requested of all community members:
+* To participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+* To exercise consideration and respect in your speech and actions.
+* To attempt collaboration before conflict.
+* To provide helpful contributions to an issue in order to encourage its progression
+* To be respectful of differing viewpoints and experiences
+* To gracefully accept constructive criticism
+* To respect the need to prioritise issues based on NV Access, user, and community priority
```
Comment (codeofdusk) [inline CODE_OF_CONDUCT.md:21]: Consider using [Oxford spelling](https://en.wikipedia.org/wiki/Oxford_spelling), as NVDA has an international userbase.

```suggestion
* To respect the need to prioritize issues based on NV Access, user, and community priority

```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## 8. Scope
+We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues--online and in-person--as well as in all one-on-one communications pertaining to community business.
+This code of conduct and its related procedures also applies to unacceptable behaviour occurring outside the scope of community activities when such behaviour has the potential to adversely affect the safety and well-being of community members. 
+For others setting up or running an NVDA group of any kind, it is strongly recommended that a complementary Citizen and Contributor Code of Conduct is adopted. 
+
+## 9. Contact info
+info@nvaccess.org
+
+## 10. License and attribution
```
Comment (codeofdusk) [inline CODE_OF_CONDUCT.md:60]: In Oxford spelling:

```suggestion
## 10. Licence and attribution

```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+The Citizen and Contributor Code of Conduct is distributed by NV Access Limited. 
+Portions of text were derived from the GitHub sample Citizen and Contributor Code of Conduct, which is distributed under a Creative Commons Attribution-ShareAlike license.
```
Comment (codeofdusk) [inline CODE_OF_CONDUCT.md:62]:
```suggestion
Portions of text were derived from the GitHub sample Citizen and Contributor Code of Conduct, which is distributed under a Creative Commons Attribution-ShareAlike licence.

```

**Review by codeofdusk — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+A supplemental goal of this Code of Conduct is to increase open source citizenship by encouraging participants to recognise and strengthen the relationships between our actions and their effects on our community. 
+Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+
+## 3. Expected Behaviour
+The following behaviours are expected and requested of all community members:
+* To participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+* To exercise consideration and respect in your speech and actions.
+* To attempt collaboration before conflict.
+* To provide helpful contributions to an issue in order to encourage its progression
```
Comment (codeofdusk) [inline CODE_OF_CONDUCT.md:18]:
```suggestion
* To provide helpful contributions to an issue to encourage its progression

```

Comment (michaelDCurran) [reply CODE_OF_CONDUCT.md:18]: Thanks for the feedback @codeofdusk .
As the document itself is essentially a policy set forth by NV Access Limited, changes to the actual text itself cannot be addressed here. This pr is merely to add the document (as is) to the NVDA repository, and link to it from various locations.
The document text conforms to accepted Australian English spelling rules (which I'll admit is perhaps a little loose). E.g. My Australian English spell checker does allow a z in prioritize and an s in license. However, NV Access may choose to review this text in the future, and at that stage will update the code of conduct file in the repository.

**Pull request merged**